app/src/main/java/fr/free/nrw/commons/nearby/Place.java

```diff
@@ -60,10 +60,25 @@ public static Place from(NearbyResultItem item) {
         if(!StringUtils.isBlank(itemClass)) {
             classEntityId = itemClass.replace("http://www.wikidata.org/entity/", "");
         }
+        // Set description when not null and not empty
+        String description = (item.getDescription().getValue() != null && !item.getDescription().getValue().isEmpty()) ? item.getDescription().getValue() : "";
+        // When description is "?" but we have a valid label, just use the label. So replace "?" by "" in description
+        description = (description.equals("?")
+            && (item.getLabel().getValue() != null
+            && !item.getLabel().getValue().isEmpty()) ? "" : description);
+        /** If we have a valid label
```

Comment:
Very minor detail: Since this is not Javadoc, the [convention](https://www.oracle.com/java/technologies/javase/codeconventions-comments.html) is:
```
/*
 * Bonjour
 * les amis
 */
```